src/proxy-animation.js

```diff
@@ -882,29 +882,45 @@ function autoAlignStartTime(details) {
     return;
   }
 
-  const previousRangeDuration = details.rangeDuration
+  const previousRangeDuration = details.rangeDuration;
+
+  let startOffset, endOffset;
+
 
   // 5. Let start offset be the resolved timeline time corresponding to the start of the animation attachment range.
   //    In the case of view timelines, it requires a calculation based on the proportion of the cover range.
-  const startOffset = CSS.percent(fractionalStartDelay(details) * 100)
+  try {
+    startOffset = CSS.percent(fractionalStartDelay(details) * 100);
+  } catch {
+    // Could not resolve the start offset
```

Comment:
Do we also need a TODO here, similar to the `endOffset` parsing below?

Reply:
Probably.. 👍 
Updated with separate TODOs right now. 

The TODOs really belong in parseAnimationRange/parseTimelineRangeOffset but these are going to be refactored in #153 and I don't want to make merging any harder 😅